odoo/sql_db.py

```diff
@@ -42,6 +42,7 @@ def undecimalize(value, cr):
 
 _logger = logging.getLogger(__name__)
 _logger_conn = _logger.getChild("connection")
+_logger_savepoint = logging.getLogger("odoo.savepoint.count")
```

Comment:
sql_db log is often muted during a savepoint. We need another logger to be sure see the log